README.md

```diff
@@ -1,10 +1,10 @@
-# `kcp` provides a true multi-tenant Kubernetes control plane for workloads on many clusters
+# `kcp` provides a multi-tenant Kubernetes API service and a transparent multi-cluster workload service
 
 ![build status badge](https://github.com/kcp-dev/kcp/actions/workflows/ci.yaml/badge.svg)
 
-`kcp` is a generic [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRD) apiserver that is divided into multiple "[logical clusters](docs/investigations/logical-clusters.md)" that enable multitenancy of cluster-scoped resources such as CRDs and Namespaces. Each of these logical clusters is fully isolated from the others, allowing different teams, workloads, and use cases to live side by side.
+`kcp` has two layers.  The lower layer is a generic [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRD) apiserver that is divided into multiple "[logical clusters](docs/investigations/logical-clusters.md)" that enable multitenancy of cluster-scoped resources such as CRDs and Namespaces. Each of these logical clusters is fully isolated from the others, allowing different teams, workloads, and use cases to live side by side.
```

Comment:
If I'm right, you don't _have_ to have multiple logical clusters; if that feature wasn't useful for you, you could use KCP without paying any attention to the logical cluster aspect at all.

If so:
```suggestion
`kcp` has two layers.  The lower layer is a generic [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRD) apiserver that you can further divide into multiple "[logical clusters](docs/investigations/logical-clusters.md)". KCP logical clusters enable multitenancy for cluster-scoped resources such as CRDs and Namespaces. Each of these logical clusters is fully isolated from the others, allowing different teams, workloads, and use cases to live side by side.
```